chat-plugins/mafia.js

```diff
@@ -1231,7 +1235,7 @@ class MafiaTracker extends Rooms.RoomGame {
 			// if there's only one option, it's their role, parse it properly
 			let roleName = '';
 			if (this.IDEA.data.picks.length === 1) {
-				const role = parseRole(player.IDEA.picks[this.IDEA.data.picks[0]]);
+				const role = MafiaTracker.parseRole(player.IDEA.picks[this.IDEA.data.picks[0]]);
```

Comment:
Again, `this`

Reply:
None of these need `this`; it's a static function, meaning you have to reference the class itself, not its instance.